docs/aqua-book/aqua-js-api.md

```diff
@@ -0,0 +1,64 @@
+# Aqua JS API
+
+Aqua JS API allows you to manage all aspects of [Aqua](../introduction.md) development from your JS projects.
+
+To install the Aqua CLI package:
+
+```sh
+npm install --save @fluencelabs/aqua-api
+```
+
+# API
+API turns high-level Aqua code into AIR language that can be used with Fluence JS client to make peer-to-peer calls.
+
+API has only one function:
+
+```typescript
+export class Compiler {
+    compile(input: Input | Path | Call, imports: string[], config?: AquaConfig): Promise<CompilationResult>;
+}
+```
+
+where input can be:
+
+- Aqua code as a string
+
+```typescript
+export class Input {
+    input: string
+}
+```
+
+- Path to `.aqua` file
+
+```typescript
+export class Path {
+    path: string
```

Comment:
I think there should be some info on what paths are accepted. Absolute? relative? Why single `.aqua` file? What to do if I have several Aqua files?

How does it relate to `import`? does it look for aqua sources around `path`?